ci/scripts/python_wheel_macos_build.sh

```diff
@@ -55,7 +55,7 @@ pip install \
   --only-binary=:all: \
   --target $PIP_SITE_PACKAGES \
   --platform $PIP_TARGET_PLATFORM \
-  -r ${source_dir}/python/requirements-wheel-build.txt
+  -r ${source_dir}/python/requirements-wheel-build.txt \
```

Comment:
```suggestion
  -r ${source_dir}/python/requirements-wheel-build.txt
```

This was causing it to think "install" on the next line was also a package to install, that caused the error